### ISSUE TITLE
[FAQ] Part of the content can be shift to Events/Test Events section

### ISSUE THREAD

**Issue by gocadimic** [OpenMobileAlliance/oma-knowledge-base#162]
@jpradocueva,

FAQ section could end with the Collaborate alert.

The rest of the content from the OmaSecWorks/Get Involved/FAQ page, started with "Prerequisites for a TestFest," could be shifted to Events/Test Events as a new section, `About Test Events.`
 
 
 We can discuss this at today's meeting.

**Comment by jpradocueva** [comment]
- [ ] Remove F.A.Q from "UTILITIES" http://localhost:3000/utilities/market-potential/faq

**Comment by jpradocueva** [comment]
@gocadimic is this resolved? Are you suggesting any further changes?

**Comment by gocadimic** [comment]
@jpradocueva, yes, this is resolved. We agreed on another solution in the meantime. Thank you for your understanding and cooperation!

**Comment by gocadimic** [comment]
@jpradocueva, yes, this is resolved. In the meantime, we agreed on another solution.

**Comment by gocadimic** [comment]
@jpradocueva, yes, this is resolved. In the meantime, we agreed on another solution.
